test/e2e/tests/desk/documentTypeListContextMenu.spec.ts

```diff
@@ -1,51 +1,63 @@
 import {expect} from '@playwright/test'
 import {test} from '@sanity/test'
 
-const SORT_KEY = 'structure-tool::author::sortOrder'
-const LAYOUT_KEY = 'structure-tool::author::layout'
+const SORT_KEY = 'studio.structure-tool.sort-order.author'
+const LAYOUT_KEY = 'studio.structure-tool.layout.author'
 
 //we should also check for custom sort orders
-test('clicking sort order and direction sets value in storage', async ({page}) => {
+test('clicking sort order and direction sets value in storage', async ({page, sanityClient}) => {
   await page.goto('/test/content/author')
   await page.getByTestId('pane').getByTestId('pane-context-menu-button').click()
   await page.getByRole('menuitem', {name: 'Sort by Name'}).click()
-  const localStorage = await page.evaluate(() => window.localStorage)
-
-  expect(localStorage[SORT_KEY]).toBe(
-    '{"by":[{"field":"name","direction":"asc"}],"extendedProjection":"name"}',
-  )
+  const nameResult = await sanityClient.request({
+    uri: `/users/me/keyvalue/${SORT_KEY}`,
```

Comment:
🤔 probably need to set version to `vX` to fix the failing tests?

Reply:
Do the e2e tests in CI take place in staging?

Reply:
no they are against production

Reply:
Thought so! Can certainly change the API version if we want to, but that'll be (hopefully) be a short lived change (and won't currently fix tests 😅 )